### ISSUE TITLE
Can't view incoming directory anymore

### ISSUE THREAD

**Issue by jeremyeder** [distributed-system-analysis/pbench#230]
The pbench-unpack-tarballs script is writing an index.html which makes it impossible to view the list of results directories.

root@perf-infra: /home/pbench # grep -r Processing *
bin/pbench-unpack-tarballs:    echo "Processing - please wait..." > $incoming/index.html
root@perf-infra: /home/pbench # rpm -qa|grep pbench
pbench-sysstat-11.2.0-1.x86_64
pbench-web-server-0.3-0g7994890.noarch
pbench-agent-0.37-1g2139799.noarch

**Comment by ndokos** [comment]
I did some hardening of pbench-unpack-tarballs on perf-infra to try to catch it in the act and avoid this case. There were also some complaints in the log file about bad tests: I think I fixed those as well.
I'll keep watching until a new tarball comes in and make sure it makes it all the way through without incident.

BTW, there *is* a results directory on that server, so you *should* be able to replace "incoming" with "results" in your URLs. Let me know if that's not the case.
